inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/heartbeat/HeartbeatManager.java

```diff
@@ -56,6 +56,7 @@ public class HeartbeatManager implements AbstractHeartbeatManager {
 
     public static final String DEFAULT_CLUSTER_TAG = "default_cluster";
     public static final String DEFAULT_CLUSTER_NAME = "default_dataproxy";
+    public static final String DEFAULT_CLUSTER_EXT_TAG = "default=true";
```

Comment:
Why `default=true`?

Reply:
k=v is the format for extTag
this default value is also used at manager